engine/language-server/src/main/scala/org/enso/languageserver/effect/Exec.scala

```diff
@@ -18,7 +17,7 @@ trait Exec[-F[_, _]] {
     * @param op effect to execute
     * @return a future containing either a failure or a result
     */
-  def exec[E, A](op: F[E, A]): Future[Either[E, A]]
+  def exec[E, A](op: F[E, A])(implicit trace: Trace): Future[Either[E, A]]
```

Comment:
We seem to have documentation for the other parameters, so probably `trace` would be good as well (and all the other methods defining that parameter)